lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ineligible.yaml

```diff
@@ -51,14 +51,11 @@ CRI_TICF_BEFORE_RETURN_TO_RP:
     next:
       targetState: RETURN_TO_RP
     enhanced-verification:
-      targetJourney: FAILED
-      targetState: FAILED_NO_TICF
+      targetState: RETURN_TO_RP
     alternate-doc-invalid-dl:
-      targetJourney: FAILED
-      targetState: FAILED_NO_TICF
+      targetState: RETURN_TO_RP
     alternate-doc-invalid-passport:
-      targetJourney: FAILED
-      targetState: FAILED_NO_TICF
+      targetState: RETURN_TO_RP
     fail-with-ci:
```

Comment:
Should this be changed as well? So any CI will still return you to the RP

Reply:
Changed to -return_to_rp- for following events; 
alternate-doc-invalid-passport/alternate-doc-invalid-dl/enhanced-verification